### PR TITLE
Feature/fc taggingfrontend

### PR DESCRIPTION
**Pull request by bpalomino5**
Hey team! Let's go through these 97 commits! And clean this up

### CODE REVIEW

File: src/components/project-page/ProjectPage.jsx [hunk 1]
```diff
@@ -24,21 +26,30 @@ const LoadingView = () => (
...
-  state = { isEditing: false, project: null, isOwner: false, pid: null };
+  state = { isEditing: false, project: null, editable: false, pid: null };
 
   componentDidMount = async () => {
-    const { location } = this.props;
-    const values = queryString.parse(location.search);
-    const project = await db.getProjectByID(values.id);
-    const isOwner = auth.isCurrentAuthUser(project.owner);
-    this.setState({ project, isOwner, pid: values.id });
+    // eslint-disable-next-line react/destructuring-assignment
```
[the remark this answers is not in the document]
Comment (sarimabbas) [reply src/components/project-page/ProjectPage.jsx]: Doing this in https://github.com/thinkspaces/thinkspaces-react/tree/feature/sa-convert-ProjectPage-hooks

File: src/components/explore-projects/components/filter-options/FilterOptions.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          toggle={() => toggle('toggleCommitment')}
+          onSelect={onSelect}
+          filterItems={commitments}
+        />
+      </Row>
+      <hr style={{ margin: 5 }} />
+    </div>
+);
+
+export default FilterOptions;
```
[the remark this answers is not in the document]
Comment (feliciachang) [reply src/components/explore-projects/components/filter-options/FilterOptions.jsx]: yep! will be working to make filter and filter options reusable. going to try to integrate the filtering with the tag-bucket backend. see code in feature/fc-shared-filter-component

File: src/components/project-page/components/project-dashboard/ProjectDashboard.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      <div className={styles.header}>
+        <h2>Project Dashboard</h2>
+        <button type="button" className={styles.external} onClick={close}>
+          <span>View project</span>
+          <FontAwesomeIcon icon="external-link-alt" />
+        </button>
+      </div>
+      <div className={styles.dashboard}>
+        <div className={styles.sidebar}>
+          <button type="button" name="description" className={styles.sidebarItem} onClick={handleSidebar}>Description</button>
```
[the remark this answers is not in the document]
Comment (bpalomino5) [reply src/components/project-page/components/project-dashboard/ProjectDashboard.jsx]: I'm handling this one 👍

File: src/components/project-page/ProjectPage.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarimabbas) [reply src/components/project-page/ProjectPage.jsx]: Ready for review in #22

**Pull request merged**